pyproject.toml

```diff
@@ -89,7 +89,7 @@ rocketchat_API = ">=0.6.31,<1.16.0"
 colorhash = "~1.0.2"
 jsonschema = "~3.2"
 packaging = ">=20.0,<21.0"
-pytz = ">=2019.1,<2021.0"
+pytz = ">=2019.1,<2022.0"
```

Comment:
this doesn't match what the title says: "Bump pytz from 2020.5 to 2021.1"
Also there doesn't seem to be a version `2022.0` https://pypi.org/project/pytz/#history

@alwx any reason this would happen?

Reply:
ah ok it's because it uses strict less than `<`